RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -0,0 +1,10 @@
+def esproducers_by_type(process, *types):
+    return (module for module in process._Process__esproducers.values() if module._TypedParameterizable__type in types)
+def customizeMinPtForHitRecoveryInGluedDet(process,value):
+   for esp in esproducers_by_type(process, "Chi2MeasurementEstimatorESProducer", "Chi2ChargeMeasurementEstimatorESProducer"):
+       esp.MinPtForHitRecoveryInGluedDet = value
```

Comment:
please change this to
```python
       esp.MinPtForHitRecoveryInGluedDet = cms.double(value)
```
so that this customization applies as is to HLT and possibly other PSets that were not derived from a clone.
If done, also add
```python 
import FWCore.ParameterSet.Config as cms
```
at the top

With these modifications the function can be tested with ```--customise RecoTracker/Configuration/customizeMinPtForHitRecoveryInGluedDet.customizeHitRecoveryInGluedDetOff``` passed to the cmsDriver.py or runTheMatrix.py